experiment/simulation/index.html

```diff
@@ -1,13 +1,45 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-	<!-- Add CSS at the head of HTML file -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css">
```

Comment:
Bulma needs to be removed. I know you are already in touch with Shlok about it. Keeping this comment for reminder.